website/content/labs/advanced-git.md

```diff
@@ -495,106 +495,88 @@ includes a line telling us which two commits were merged:
+https://github.com/uchicago-cs/2020-advanced-git
 
 Remember that we don't have push access on that repository, but we do have
-push access on our fork. In fact, a common reason for creating a fork of
-a repository is so we can make some changes to that fork, and then contribute
-our changes back to the original repository.
-
-The way we do this is by doing a *pull request*. In other words, we submit a request
-to the original repository to pull our changes. You are going to create a pull request
-with the changes you made in your fork. To do this, go to your forked repository on GitHub:
-
-https://github.com/GITHUB_USERNAME/cmsc22000-dummy-repo
-
-You will see a "New pull request" button above the file listing. Click it. The next
-page will summarize the commits you are going to include in the pull request. If
-you see anything that looks amiss, please ask on Piazza. Otherwise, click on the
-green "Create pull request" button. You will now be able to specify a title and
-a summary for the pull request. For example, in this case, the title could be "Adding author
-to echo.c" and the summary could be "I am the author of this file". Finally, click
-"Create pull request" again, and this will create the pull request. You can verify
-that your pull request was submitted to the original repository here:
+push access on our fork. Now that we've made some changes in our fork,
+we may want to ask the owner of the original repository to merge our changes
+into the repository we forked from.
 
-https://github.com/uchicago-cs/cmsc22000-dummy-repo/pulls
-
-Now, once we receive your pull request, we will actually deny it (you're not the author of echo.c!),
-but you will have a chance to submit a successful pull request in the next section.
-
-# Code reviews with pull requests
-
-In the example we have seen so far, you forked a repository so you could make
-changes to it and submit those changes back to the original repository. This
-is a very common collaboration pattern, specially in open source software.
+This is a very common collaboration pattern, specially in open source software.
 If you spot a bug in your favorite software project, and their code is hosted
 on GitHub, you can create a fork of that repository, create a bugfix, and
-submit it back to the original repository using a pull request.
+submit it back to the original repository.
 
-In the class project, however, you will not need to create a fork of your
-team's repository, because you *do* have push access in that repository,
+The way we do this is by doing a *pull request*. 
+In the class project, however, you will not need to create a fork of the
+chiventure repository, because you *will* have push access in that repository,
 but with an important restriction: you are allowed to create additional
 branches and push to them, but you are *not* allowed to push to the `master`
 branch (in this case, we say that `master` is a *protected branch*). 
```

Comment:
With the new developer guidelines, won't `dev` be the branch they're creating PR's for? (and would therefore be protected as well?)

Reply:
Added a note clarifying this.